src/refs.c

```diff
@@ -1130,6 +1130,18 @@ int git_reference__update_terminal(
 	return error;
 }
 
+const char *git_reference__commit_type(const git_commit *commit)
```

Comment:
This function is used inside of this file only and will probably never be exposed to other callers, as it is very specific. As such, this function should be marked as `static` as well as drop the `git_reference__` prefix.